src/main/java/core/mate/academy/service/producer/ExcavatorProducer.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.service.producer;
+
+import core.mate.academy.model.Excavator;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {
```

Comment:
Don't begin class or method implementation with an empty line. Remove the empty line at the beginning of the class definition.